src/_posts/platform/getting-started/2000-01-01-how-to-migrate-from-heroku.md

```diff
@@ -5,43 +5,78 @@ tags: heroku tutorial
+Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). This page is here to help your migrate from Heroku to Scalingo as easir as possible.
 
-It is so easy that you can use [heroku2scalingo](https://github.com/Scalingo/heroku2scalingo/releases/latest), a tool to migrate a Heroku app to Scalingo.
+## Requirements
 
-## heroku2scalingo
+To be able to follow this tutorial, we assume that you have:
 
-### Usage
+- A Scalingo account
+- The Scalingo [CLI]({% post_url platform/cli/2000-01-01-start %}) installed
+- A local copy of the repository containing your app’s code
+
+Please ensure you are running all commands above from the root of the repository containing your app’s code. 
 
 ```bash
-./heroku2scalingo <herokuAppName>
+$ cd <app directory>
 ```
 
-It works by performing the following operations in the same order as below:
+Note: All operations described below can also be done through our web dashboard (instead of the command line tool). 
+
+## Migration Steps
 
-* Authentication to Scalingo
-* Authentication to Heroku API
-* Creation of Scalingo app
-* Get/Set environment variables
-* `git clone` your Heroku app repository
-* `git push scalingo master` -> Auto-deployment using the `Procfile`
+### Creating your Scalingo app
 
-### Notes
+- Start by login into the Scalingo platform using our `cli` :
 
-Other advanced operations are planned, for more informations take a look at [the GitHub project](https://github.com/Scalingo/heroku2scalingo/#todo).
+```bash
+$ scalingo login
+```
 
-### Release
+- You can now create your app : 
+
+```bash
+# Replace <my-app> by your actual app name
+$ scalingo create <my-app>
+```
 
-You can download the latest version by following this link : [https://github.com/Scalingo/heroku2scalingo/releases/latest](https://github.com/Scalingo/heroku2scalingo/releases/latest).
+### Configure Environment Variables
 
-## Advanced apps migration
+- First retrieve the environment variables from your current Heroku app context :
 
-* The [Procfile]({% post_url platform/app/2000-01-01-procfile %}) can remain the same
-* You need to set the same environment variables as the ones defined on your Heroku app
-* You have to set your Scalingo addons according to the addons you had on your Heroku app.
-  If you require an addon that is not present in [this list](https://scalingo.com/addons),
-  feel free to [send us an email](mailto:support@scalingo.com).
-* You will need to migrate any database manually, dumping it from Heroku and restoring it to Scalingo:
+```bash
+$ heroku config
+```
+
+- Those environment variables need to be declared in your Scalingo app context. For instance, this command will set the `NODE_ENV` environment variable:  
+
+```bash
+# Replace <my-app> by your actual app name
+$ scalingo -a <my-app> env-set NODE_ENV=production
```

Comment:
- I would remove the comment as you already explained that in the previous code block
- Documentation should use long option name (`--app`) instead of the short one (`-a`)
- I would keep consistency with other doc pages and use `my-app` rather than `<my-app>`

```suggestion
$ scalingo --app my-app env-set NODE_ENV=production
```